frb_example/with_flutter/macos/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :osx, '10.11'
+platform :osx, '10.14'
```

Comment:
hmm if the old version is good, maybe we should use the old one for more compatibility?